### ISSUE TITLE
🤩 Introducing Wavebox 10

### ISSUE THREAD

**Issue by Thomas101** [wavebox/waveboxapp#1133]
Today we are delighted to announce the release of a brand new version of Wavebox. The app has been completely rebuilt from the ground up, on a full Chromium foundation to provide a flawless experience across all your web apps.

To find out why we’ve made the move from Electron to Chromium, [read this blog post](https://blog.wavebox.io/wavebox-is-evolving-electron-chromium/)

We’re very excited about the release of Wavebox 10 and the potential it has for offering fast and seamless working across your web apps and tools. It's a huge step forward from Wavebox Classic (the new name for existing Wavebox) and we hope you will find it leaner, more robust and much faster than ever before.

Here's a really brief list of what's new in Wavebox 10...

* Expand our supported extension list from 16 to thousands of extensions ❤️ 
* A 10-20% decrease in memory consumption 💨
* A refined and sparkly new UI that's tastefully familiar ✨
* Seamless tab support in any window
* Fully integrated autofill for usernames and passwords 🤖
* Improved PDF and printing support across the board
* Improved theming support 🎨
* Support for things such as U2F keys when signing into Google
* Improved screen-sharing support
* Privacy lock to keep your app private 🔒

...and if you want to download it, **just head over to our [download page](https://wavebox.io/download)**.

If you're already on the beta channel and want to continue getting new features first, then you can [download the beta directly](https://wavebox.io/download/beta)

**We're not pushing this down the normal update channels just yet, as we're yet to release our migration wizard,** which means you will have to setup a-fresh. If you don't mind doing this (or want to clear out a few cobwebs 🕸️) then the download page is for you 😊.

There's more in-depth information about everything that's new 🆕 in the [Wavebox 10 launch blog](https://blog.wavebox.io/stay_focused_be_more_productive_wavebox10)

**Comment by CheechGe** [comment]
The windows 10 version would not install.  I downloaded it, tried to do the install and nothing happened, rebooted my PC, and tried the install again with the same results.  downloaded it again (file size was the same) and retried the install again with the same results.  It did not show up in the task manager at any time.  The R4 version still works fine. 

Suggestions?

**Comment by dfrankland** [comment]
@Thomas101 Hold up. Is Wavebox 10 no longer open source? The whole reason why I use Wavebox over other competitors is that it is open source. I don't see this changed mentioned anywhere.

**Comment by bkuri** [comment]
@dfrankland seems to be true, unfortunately… #1132

**Comment by GrigorijST** [comment]
Wavebox Version: 10.0.52-2 stable (Wavebox Pro)
    Operating System & Version: Linux Mint 19.3
    (Linux only) Install method: Debian installer

Upgraded to 10.x version from 4.x and was very disappointed that:
Missing important function: close button --> hide to tray

Now Wavebox all time hangs in minimized state and Plank panel.
It is very uncomfortable! Can you add this function (as it was on the previous version of Wavebox) as soon as possible.

Thanks.

**Comment by EnorMOZ** [comment]
Wavebox 10 repo ?

**Comment by paulmicheli** [comment]
Is Wavebox 10 available on Fedora 31?

**Comment by Thomas101** [comment]
@paulmicheli yes. There's an RPM repo that you can use with Fedora 31. There's info on how to add the sources and install here https://wavebox.io/download

**Comment by yermulnik** [comment]
Is there a place where I can download older versions of Wavebox 10 rather than `latest`?
I've upgraded to `10.0.142` today from official repo `https://download.wavebox.app/core/linux/deb/ amd64/` and it started to hang my laptop. I couldn't identify the issue yet hence was going to downgrade to previous version which was `10.0.128` but it's not that obvious where do I get it (I managed to do it but I'm wondering whether there's a more straightforward approach).
Also do I file issues for Wavebox 10 in this repo or there's some other more appropriate place (as I understood this repo is for Wavebox pre-10)?

**Comment by Thomas101** [comment]
@yermulnik you can grab a tar of the previous version from https://download.wavebox.app/core/linux/tar/Wavebox_10.0.128-1.tar.gz

Can you drop support@wavebox.io some additional info, including details of which operating system you're using, some system specs and info about how and when it freezes?

**Comment by yermulnik** [comment]
@Thomas101 Thanks for the link. That is exactly what I used to download version I had installed previously. With one exception: I had `10.0.128-2` though `10.0.128-1` works just fine w/o interrupting my laptop with unexpected hangs.
I'll try my best to double check the latest release during weekend and gather as much info as possible about the issue.

**Comment by sfrique** [comment]
I installed the latest at archlinux ( 10.0.142-2  )and my system keeps crashing (the whole X server).
I don't know exactly what is happening but only does after I open wavebox for a while.

I will downgrade and see if it stop from happening.

**Comment by Thomas101** [comment]
@sfrique would you be able to drop support@wavebox.io an email with some more details about your setup. Window manager, kernel version, Xorg version (`sudo X -version`) would be really useful, as well as if you have Chrome/Chromium installed and if they exhibit the same behaviour

**Comment by sfrique** [comment]
@Thomas101 I will do it when I get a chance, but for the time being the version Wavebox Version 10.0.128.1 core didn't have any crashes or anything.
So I guess I will keep it for a while. If i try to update it again we will see!

**Comment by yermulnik** [comment]
@sfrique haven't you tried v10.0.148 yet (it's just been released though)?
I'm not quite keen to play with crashing my X system and/or freezing laptop as it influences my working hours drastically and my employer is not indeed happy with that =(
I'm wondering whether this issue might be related https://github.com/wavebox/waveboxapp/issues/1238

**Comment by sfrique** [comment]
@yermulnik I have not.
I was told by the reply on the support email,  that I should wait, more users are experiencing that.
Since the current install is working as expected, I will wait longer to update it.
Like you, I can't afford to have it crashing.

**Comment by yermulnik** [comment]
@sfrique Oh, thanks  for info dude. Would you please update this thread once you've got any further info from support? I'd appreciate this very much.

**Comment by sfrique** [comment]
@yermulnik  Will try to! but the support email was answered by @Thomas101 so he will probably update here

**Comment by yermulnik** [comment]
@Thomas101 @sfrique Hey dudes, any updates so far re the issue?

**Comment by yermulnik** [comment]
Has this been fixed in `10.0.158`?

**Comment by sfrique** [comment]
> @Thomas101 @sfrique Hey dudes, any updates so far re the issue?

I have not updated it.

**Comment by yermulnik** [comment]
@Thomas101 Any updates so far? Has the issue been fixed? I'd expect some comms at least to those who pay money...
@sfrique Haven't you tried any of latest Wavebox releases yet?

**Comment by sfrique** [comment]
@yermulnik I did not, please stop asking me for updates on this, I am hanging just like you.
The version I am using is working, so, for now, I am ok.

If I try a next version I will update it here. Please you try and let me know as well.

**Comment by yermulnik** [comment]
Thanks Henrique. Apologies for nudging.

**Comment by yermulnik** [comment]
Have just tried `10.0.172-1`. Still the same issue — X goes away restarting =(
I attached an excerpt from `/var/log/syslog` right after Wavebox start and right before X restart that seem to be related to Wavebox and the issue.
[wavebox-breaking-X.log](https://github.com/wavebox/waveboxapp/files/4705510/wavebox-breaking-X.log)

**Comment by MichaelTunnell** [comment]
so @Thomas101 please clarify if you have removed Wavebox from being open source and is now ***proprietary***?

This was already asked, I assume you missed that comment. As someone who has been a user and a paid user even since the days of WMail, I'd very much prefer an answer to this question.

**Comment by yermulnik** [comment]
@Thomas101 Would you please give me some insights on the issue I'm affected with (and not only me) or is there any other public place where I can get an answer (at least as a paid user)?

**Comment by Thomas101** [comment]
@yermulnik thanks for reaching out to support with more info - we're working on replicating the issue here so that we can get to the bottom of this

**Comment by MichaelTunnell** [comment]
@Thomas101 waiting on reply about Open Source vs Proprietary . . .

**Comment by EnorMOZ** [comment]
@MichaelTunnell https://github.com/wavebox/waveboxapp/issues/1132 👎

**Comment by sfrique** [comment]
I can confirm version `10.0.195_2-1`  from aur on archlinux still have the same problem of Xorg restarting.

**Comment by yermulnik** [comment]
@Thomas101 Hi Thomas, since I've switched to macbook temporarily where Outlook works natively I'm not able to test what you send me about a month ago (and I can't predict when I'll switch back to linux at the moment). Wouldn't you mind me sharing the link with @sfrique to see whether he's got a chance to test those builds?

**Comment by adelavega** [comment]
@MichaelTunnell kind of weird they would ignore your question. I mean nothing wrong with them choosing that, their choise; but it would make sense to address it.

**Comment by MichaelTunnell** [comment]
> @MichaelTunnell kind of weird they would ignore your question. I mean nothing wrong with them choosing that, their choise; but it would make sense to address it.

I suspect it is a "ignore them so maybe they will go away and we can avoid being massive hypocrites"
